specs2/specs2.bzl

```diff
@@ -2,87 +2,28 @@ load(
-            "specs2_fp": "e43006fdd0726ffcd1e04c6c4d795176f5f765cc787cc09baebe1fcb009e4462",
-            "specs2_matcher": "448e5ab89d4d650d23030fdbee66a010a07dcac5e4c3e73ef5fe39ca1aace1cd",
-        },
-        "2.12": {
-            "specs2_common": "7b7d2497bfe10ad552f5ab3780537c7db9961d0ae841098d5ebd91c78d09438a",
-            "specs2_core": "f92c3c83844aac13250acec4eb247a2a26a2b3f04e79ef1bf42c56de4e0bb2e7",
-            "specs2_fp": "834a145b28dbf57ba6d96f02a3862522e693b5aeec44d4cb2f305ef5617dc73f",
-            "specs2_matcher": "78c699001c307dcc5dcbec8a80cd9f14e9bdaa047579c3d1010ee4bea66805fe",
-        },
-    }
-
-    scala_version_jar_shas = scala_jar_shas[major_version]
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_org_specs2_specs2_common",
-        artifact = _scala_mvn_artifact(
-            "org.specs2:specs2-common:" + specs2_version(),
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["specs2_common"],
-        deps = ["@io_bazel_rules_scala_org_specs2_specs2_fp"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_org_specs2_specs2_core",
-        artifact = _scala_mvn_artifact(
-            "org.specs2:specs2-core:" + specs2_version(),
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["specs2_core"],
-        deps = [
-            "@io_bazel_rules_scala_org_specs2_specs2_common",
-            "@io_bazel_rules_scala_org_specs2_specs2_matcher",
+        maven_servers = _default_maven_server_urls(),
+        overriden_artifacts = {}):
+    repositories(
+        for_artifact_ids = [
+            "io_bazel_rules_scala_org_specs2_specs2_common",
+            "io_bazel_rules_scala_org_specs2_specs2_core",
+            "io_bazel_rules_scala_org_specs2_specs2_fp",
+            "io_bazel_rules_scala_org_specs2_specs2_matcher",
         ],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_org_specs2_specs2_fp",
-        artifact = _scala_mvn_artifact(
-            "org.specs2:specs2-fp:" + specs2_version(),
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["specs2_fp"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_org_specs2_specs2_matcher",
-        artifact = _scala_mvn_artifact(
-            "org.specs2:specs2-matcher:" + specs2_version(),
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["specs2_matcher"],
-        deps = ["@io_bazel_rules_scala_org_specs2_specs2_common"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
+        scala_version = scala_version,
+        maven_servers = maven_servers,
```

Comment:
If we're using the default servers all the time - why can't it be just the default value and skip the explicit parameter?

Reply:
Users may provide other server urls